core/src/main/java/com/example/AutomatedReviewTest.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import java.util.*;
+
+// The goal of this code is to find out if code analysis will automatically add
+// commentary to a pull request when it finds problems with the code.
+
+public abstract class AutomatedReviewTest extends Object {
+
+  public boolean hello(Optional unusedRaw)
+  {
+    float x = 3.0f;
+    if (x + 2.0f * 1.0f == 5.0f)
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Equality tests should not be made with floating point values. ([squid:S1244](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AS1244))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack&pullRequest=9&issues=AWY_zChlvOM6zLh9Plco&open=AWY_zChlvOM6zLh9Plco)

[ISSUE_KEY]: AWY_zChlvOM6zLh9Plco